test-lib.sh

```diff
@@ -799,7 +799,7 @@ EOF
     fi
 
     # Check if -v parameter is present in s2i_args and add it into docker build command
-    read -ra mount_options <<< "$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*' || true)"
+    read -d'\n' -ra mount_options <<< "$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*' || true)"
```

Comment:
`-d'\n'` is used implicitly, I thought you wanted to use `-d ''`

Reply:
I have try it and the output from both is the same:

```
$ s2i_args="-e HTTPD_LOG_TO_VOLUME=1 --user 0 -v /tmp/dir1:/tmp/dir2 -v /var/dri:/var/dri2"
$ read -d '' -ra mount_options <<< "$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*' || true)"
$ echo ${mount_options[@]+"${mount_options[@]}"}
-v /tmp/dir1:/tmp/dir2 -v /var/dri:/var/dri2
```

```
$ s2i_args="-e HTTPD_LOG_TO_VOLUME=1 --user 0 -v /tmp/dir1:/tmp/dir2 -v /var/dri:/var/dri2"
$ read -d '\n' -ra mount_options <<< "$(echo "$s2i_args" | grep -o -e '\(-v\)[[:space:]]\.*\S*' || true)"
$ echo ${mount_options[@]+"${mount_options[@]}"}
-v /tmp/dir1:/tmp/dir2 -v /var/dri:/var/dri2
```

Reply:
But I have replaced '\n' with ''

Reply:
The output indeed does not change in either of those cases. This is interesting and leads me to believe that using `-d '\n'` does not actually make `read` use newline as a delimiter.
In any case we do not want to use something that is not entirely obvious from the code (I would expect only a single line to be read when `-d '\n'` is used), so changing it to `-d ''` is worthwhile anyway. Thanks!